### ISSUE TITLE
Timed Holiday Greetings build up

### ISSUE THREAD

**Issue by Shahelion** [HenryLoenwind/EnderIOAddons#27]
If you have the game paused in singleplayer, they build up. Resulting in an extremely laggy storm of fireworks the second you unpause it.
![2016-01-01_01 07 01](https://cloud.githubusercontent.com/assets/6917535/12070356/1bdfc25e-b024-11e5-8e0e-04552a80a07e.png)

**Comment by HenryLoenwind** [comment]
I was more concerned about server lag than singleplayer when I made this, tbh. I'll have a look at this before next New Year's ;)

**Comment by Shahelion** [comment]
Is it only for new years?

**Comment by HenryLoenwind** [comment]
Yes, the fireworks are only for New Year's. Christmas has free energy with snow and multi-language season greetings on the RF bars. There's not yet a Easter egg for Easter ;)

**Comment by Shahelion** [comment]
Status?
